Introduce test for emptyepsilon

r-richardson opened the pull request:
This PR introduces a test checking emptyepsilon's ability to start

- Related Progress Issue: https://progress.opensuse.org/issues/176172
- Verification run: https://openqa.opensuse.org/tests/5138881

r-richardson committed:
Introduce test for emptyepsilon

This PR introduces a test checking emptyepsilon's ability to start

Related Progress Issue: https://progress.opensuse.org/issues/176172

Co-authored-by: gpuliti <gpuliti@suse.com>
Co-authored-by: baierjan <jbaier@suse.com>

okurz commented:
Also at best add the dependency for `perl(Inline::Perl)` like in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22508#pullrequestreview-2971876603

r-richardson committed:
Fix lua test file recognition in yaml schedules

Allows for .lua files to be properly recognized and evaluated within yaml
schedules

Added as part of https://progress.opensuse.org/issues/176172

Co-authored-by: gpuliti <gpuliti@suse.com>
Co-authored-by: baierjan <jbaier@suse.com>

r-richardson committed:
Add Inline::Lua for osado already Lua tests

Add dependency as in
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22508
but for Lua code

Part of https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/22505

dcermak reviewed (commented):
A third language? Why?

okurz commented:
> A third language? Why?

Because it's a demonstrator showing how it's possible :) Given how many people are scared of Perl or have the misconception that "openQA tests need to be done in Perl hence I will not contribute" we wanted to show alternatives

@r-richardson need to rebase

schlad reviewed (changes requested):
Please amend the git commit "Introduce test for emptyepsilon" with more descriptive info of what you are tying to do
